src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SourceBuiltArtifactsTests.cs

```diff
@@ -20,6 +20,7 @@ public class SourceBuiltArtifactsTests : SdkTests
     [Fact]
     public void VerifyVersionFile()
     {
+        Assert.NotNull(Config.SourceBuiltArtifactsPath);
```

Comment:
IMO this yields a bad UX for when a configuration setting is missing.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/54ec6057312e22493f8f8ffc00dae655e0dcb385